src/ui/common/src/components/pages/workflows/index.tsx

```diff
@@ -191,7 +215,6 @@ const WorkflowsPage: React.FC<Props> = ({ user, Layout = DefaultLayout }) => {
           skip: dagResultsLoading && latestDagId,
```

Comment:
I feel this is probably wrong as it will not skip if `dagResultsLoading` is false but `latestDagId` is empty. I'd just use `skip: !latestDagId` to load dag whenever the ID is available.